pillar.example

```diff
@@ -13,6 +13,9 @@ bind:
     options:
       allow-recursion: '{ any; };'  # Never include this on a public resolver
 
+   # force bind to serve only one protocol. omitting this reverts to binds default of both.
```

Comment:
Can we update this line to say something like `force bind to serve only one IP protocol (ipv4: 4, ipv6: 6)...`